rust/src/mask/config/mod.rs

```diff
@@ -164,16 +164,18 @@ pub struct MaskConfig {
 }
 
 impl MaskConfig {
-    /// Return the number of bytes needed to represent a mask element
-    /// with this configuration
-    pub fn bytes_per_digit(&self) -> usize {
+    /// Returns the number of bytes needed for an element of a mask or masked model wrt this mask
```

Comment:
Several remarks:

1. In the code we use `MaskObject` to something that is either a mask or a masked model. Maybe we can do the same in the documentation?

2. I find that "to represent" from the sentence makes it a little less clear personally, but maybe it's just me

3. Unrelated to this PR but I wonder whether this should be `pub` actually: it really is just an implementation detail.

Reply:
ah and :+1: for the `s/digit/number`

Reply:
some questions on the remarks:
1. i currently use the phrasing "mask or masked model"  to express this dual use of `MaskObject`. do you mean the same or that it should be replaced with "mask object"?
2. not sure what you mean here, since the less clear formulation is part of the old version, whereas it doesn't appear in the new version.

i agree on making 3. private.

Reply:
since i'm not sure how to proceed on this issue, i'll leave it as "mask or masked model" for now and it can be changed later on if needed.

Reply:
just to leave another idea for later: we might even make the `MaskObject` a trait and then have distinct structs for `Mask` and `MaskedModel` to avoid type confusion.

Reply:
personally i prefer "mask object" to the wordier "mask or masked model". but i'd be happy for this to be deferred.

Reply:
i'll change it to "mask object". if we implement changes to the `MaskObject` later on, we can change the docs anyways.